opentelemetry-dotnet-contrib.proj

```diff
@@ -4,6 +4,9 @@
     <SolutionProjects Include="**\*.csproj" />
     <TestProjects Include="test\**\*.csproj" />
     <PackProjects Include="src\**\*.csproj" />
+
+    <!-- Do not pack SemanticConventions project for now -->
+    <PackProjects Remove="src\OpenTelemetry.SemanticConventions\OpenTelemetry.SemanticConventions.csproj" />
```

Comment:
nit: Remove this. If you want to exclude `OpenTelemetry.SemanticConventions` from being packaged just add `<IsPackable>false</IsPackable>` in `OpenTelemetry.SemanticConventions.csproj`.